src/lib/assetBridger/ethBridger.ts

```diff
@@ -164,6 +178,45 @@ export class EthBridger extends AssetBridger<
     return L1TransactionReceipt.monkeyPatchEthDepositWait(tx)
   }
 
+  /**
+   * Deposit ETH from L1 onto a different L2 address
+   * @param params
+   * @returns
+   */
+  public async depositTo(
+    params: EthDepositToParams | L1ToL2TxReqAndSigner,
+    l2Provider: Provider
+  ): Promise<L1ContractCallTransaction> {
+    await this.checkL1Network(params.l1Signer)
```

Comment:
Should we also check the l2 provider?

Reply:
Yes, definitely a good idea to avoid weird behaviours. It is used while creating the retryable ticket to get the Inbox contract address.